inc/views/pluggable/pagination.php

```diff
@@ -143,6 +144,116 @@ public function filter_localization( $data ) {
 		return $data;
 	}
 
+	/**
+	 * Filter the pagination links to decide whether or not to show the "Jump to Page" input field.
+	 * 
+	 * @param mixed $markup the pagination links from the filter.
+	 * 
+	 * @return mixed
+	 */
+	public function maybe_add_jump_to_page_input( $markup ) {
+
+		if ( get_theme_mod( 'neve_pagination_type', 'number' ) === 'jump-to-page' ) {
+			return $this->normalize_jump_to_input();    
+		}
+
+		return $markup;
+
+	}
+
+	/**
+	 * Create jump to navigation inputs.
+	 * 
+	 * @return mixed $markup HTML to output on page.
+	 */
+	private function create_jump_to_html() {
+
+		global $wp, $wp_query;
+
+		if ( empty( $wp->request ) ) {
+			return;
+		}
+
+		$request = '/' . $wp->request;
+
+		$button_text = apply_filters( 'neve_pagination_jump_button_text', __( 'Go', 'neve' ) );
```

Comment:
@UVLabs @cristian-ungureanu what do you think about using `greater than` sign instead of text for the button, i.e `&gt;` to avoid adding a new translation.